### PR TITLE
Fixed hover panel resizing issue for pie and donut charts 

### PR DESCRIPTION
**Pull request by dlabrecq**
Fixed hover panel resizing issue for pie and donut charts  -- parent container was too small.
In addition to the test pages, this helps chart templates display properly on the new patternfly-org site.

screen shot 2016-05-05 at 11 16 09 am
screen shot 2016-05-05 at 11 15 53 am

### CODE REVIEW

File: dist/css/patternfly-additions.css [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
+  .form-search .combobox-container,
+  .form-inline .combobox-container {
+    display: inline-block;
+    margin-bottom: 0;
+    vertical-align: top;
+  }
+  .form-search .combobox-container .input-group-addon,
+  .form-inline .combobox-container .input-group-addon {
+    width: auto;
+  }
```
Comment (jeff-phillips-18) [inline dist/css/patternfly-additions.css:12]: How is this change related?

**Comment by dtaylor113** [comment]
LGTM

File: dist/css/patternfly-additions.css (hunk 1 above)
Comment (dlabrecq) [inline dist/css/patternfly-additions.css:12]: I don't believe this is related to my changes. When I create a clean workspace (omitting any of my changes) and build, this is what I get.

I see less/lib/c3/c3.css and less/lib/bootstrap-combobox/combobox.less have a bunch of changes. I'm guessing this is because we relaxed our dependency versions?

**Comment by jeff-phillips-18** [comment]
gotcha. LGTM

**Comment by dtaylor113** [comment]
Thanks, and I appreciate the screenshots :-)
